orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/services/LogoutServiceTest.java

```diff
@@ -226,13 +250,16 @@ void successfullyReturnsErrorLogoutResponse() throws URISyntaxException {
         assertThat(response, hasStatus(302));
         ErrorObject errorObject =
                 new ErrorObject(OAuth2Error.INVALID_REQUEST_CODE, "invalid session");
-        URIBuilder uriBuilder = new URIBuilder(DEFAULT_LOGOUT_URI);
-        uriBuilder.addParameter("error_code", errorObject.getCode());
-        uriBuilder.addParameter("error_description", errorObject.getDescription());
-        URI expectedUri = uriBuilder.build();
         assertThat(
-                response.getHeaders().get(ResponseHeaders.LOCATION),
-                equalTo(expectedUri.toString()));
+                response,
+                isRedirectTo(
+                        allOf(
+                                baseUri(DEFAULT_LOGOUT_URI),
+                                queryParameters(hasEntry("error_code", errorObject.getCode())),
+                                queryParameters(
+                                        hasEntry(
+                                                "error_description",
+                                                errorObject.getDescription())))));
```

Comment:
This is really neat, and I like the syntax. I think it might be nicer to assert that `FrontEndPages.whatever` is called with the correct params, unit test that separately, and just assert that the redirect is to a static value here?

Reply:
Done